packages/widget/src/i18n/en.json

```diff
@@ -69,7 +69,7 @@
     "message": {
       "autoRefuel": "You're low on {{chainName}} gas. By continuing, you'll get enough gas to complete the transaction.",
       "emptyActiveTransactions": "Transactions in progress will appear here. Once completed, find them in transaction history.",
-      "emptyTokenList": "We couldn't find tokens on {{chainName}} chain or you don't have any. Please try search again or choose another chain.",
+      "emptyTokenList": "We couldn't find tokens on {{chainName}} chain or you don't have any. Please search by contract address if your token doesn't appear or choose another chain",
```

Comment:
> We need edit “Please try search again or choose another chain” to 
> ”Please  search by contract address if your token doesn't appear or choose another chain”

Just replaced the second part of that message.

Reply:
![image](https://github.com/lifinance/widget/assets/43956540/93c24eee-fc36-4aa7-8106-a44cd115655b)